Wiki/zh-Tw.js

```diff
@@ -0,0 +1,678 @@
+exports.LANGUAGE = {
+  CODE: "zh-Tw",
+  STANDARD_CODE: "zh-Tw",
+  FULL_CODE: "正體中文",
+  WIKI_CODE: "zh-Tw",
+  CONTRIBUTORS: ["Kami-Sqmf", "iLxlo_"]
+};
+
+exports.TIME = {
+  SECOND: "秒",
+  SECONDS: "秒",
+  MINUTE: "分",
+  MINUTES: "分",
+  HOUR: "小時",
+  HOURS: "小時",
+  DAY: "天",
+  DAYS: "天",
+  WEEK: "週",
+  WEEKS: "週",
+  MONTH: "月",
+  MONTHS: "月",
+  YEAR: "年",
+  YEARS: "年"
+};
+
+exports.PAGE_TITLES = {
+  HOME: "首頁",
+  INVITE_FILO: "邀請 Filo",
+  ANTI_BAD_WORDS: "髒話過濾",
+  ANTI_EVASION: "處罰者破解偵測",
+  ACTION_LIST: "事項清單",
```

Comment:
I don't know if this is wrongly translated, if it is well translated you can leave it as it was (the translator does not translate it to me as "List of Articles").

It should be "Action List".